integration-tests/src/test/java/tachyon/master/JournalIntegrationTest.java

```diff
@@ -157,7 +157,7 @@ public final void after() throws Exception {
   @Before
   public final void before() throws Exception {
     mLocalTachyonCluster = new LocalTachyonCluster(Constants.GB, 100, Constants.GB);
-    MasterContext.getConf().set(Constants.MASTER_JOURNAL_LOG_SIZE_BYTES_MAX,
+    mLocalTachyonCluster.getTestConf().set(Constants.MASTER_JOURNAL_LOG_SIZE_BYTES_MAX,
```

Comment:
I prefer this style when only 1 entry needs to be set, could you make it consistent for the other cases?